.rdmanifest

```diff
@@ -5,7 +5,6 @@ check-presence-script: '#!/bin/bash
   test -d $BITBOTS_CATKIN_WORKSPACE/src/humanoid_base_footprint'
 depends:
 - bitbots_docs
```

Comment:
bitbots_docs is probably not needed anymore for ros-sports

Reply:
Can we delete `.rdrmanifest`? Is it still used?

Reply:
I also have never worked with the CI. Only @ftsell and @timonegk know about it

Reply:
Discussion moved to #25, not going to remove for this PR.